### PR TITLE
one-click-buy: update cardId config position

### PR DESCRIPTION
**Pull request by allangrds**
## Description
Change the position of `cardId` on `customerParameters`.

Closes https://github.com/pagarme/mercurio/issues/271

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: public/assets/js/customParameters.js [hunk 1]
```diff
@@ -102,6 +102,7 @@ const pagarmeApi = `var configs = {
   },
   transaction: {
     amount: 10000,
+    //cardId: 'card_cjjr96qt401k5236eaqvhpvpf',
```
Comment (thiagommedeiros) [inline public/assets/js/customParameters.js]: @augusto-jm-amaral só pra vc entender, decidimos deixar comentado lá no editor só pra a gente nao esquecer desse campo onde ele deve ficar.

Comment (augusto-jm-amaral) [reply public/assets/js/customParameters.js]: Beleza Haha.

File: src/store.js [hunk 2]
```diff
@@ -24,7 +23,7 @@ export default ({
     reducers,
     {
       creditCard: {
-        cardId: path(['cardId'], configs),
+        cardId: path(['cardId'], transaction),
```
Comment (augusto-jm-amaral) [inline src/store.js]: Usa `prop` quando não é um caminho @allangrds

Comment (allangrds) [reply src/store.js]: Total mano :)

**Pull request merged**